website/docs/r/admin_organization_settings.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_admin_organization_settings"
+sidebar_current: "docs-resource-tfe-organization-x"
+description: |-
+  Manages admin settings for an organization (Terraform Enterprise Only).
+---
+
+# tfe_admin_organization_settings
+
+Manage admin settings for an organization. This resource requires the
+use of an admin token and is for Terraform Enterprise only. See example usage for
+incorporating an admin token in your provider config.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+
+provider "tfe" {
+  hostname = var.hostname
+  token    = var.token
+}
+
+provider "tfe" {
+  alias    = "admin"
+  hostname = var.hostname
+  token    = var.admin_token
+}
+
+resource "tfe_organization" "a-module-producer" {
+  name  = "my-org"
+  email = "admin@company.com"
+}
+
+resource "tfe_organization" "a-module-consumer" {
+  name  = "my-other-org"
+  email = "admin@company.com"
+}
+
+resource "tfe_admin_organization_settings" "test-settings" {
+  provider                              = tfe.admin
+  organization                          = tfe_organization.a-module-producer.name
+  workspace_limit                       = 15
+  global_module_sharing                 = false
+  module_sharing_consumer_organizations = [
+    tfe_organization.a-module-consumer.name
+  ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `organization` - (Required) Name of the organization.
+* `access_beta_tools` - (Optional) True if the organization has access to beta tool versions.
```

Comment:
🤔 I noticed that the example doesn't include `access_beta_tools` and then I wondered if we try to include all arguments in our examples? 🤔

Reply:
It would be a nice habit to do so. I've offered some general guidance for some past PRs:
* Showing all the resources that may need to be created for the documented resource to be applied successfully (at least for required attributes). 
* Having at least two examples, one with required arguments and one with all optional arguments included. It is of course dependent on the resource in question -- some arguments are mutually exclusive. We should showcase different use cases that might be helpful to our customers.